table/block_based/filter_policy_internal.h

```diff
@@ -46,7 +46,9 @@ class BuiltinFilterBitsReader : public FilterBitsReader {
   virtual bool HashMayMatch(const uint64_t /* h */) { return true; }
 };
 
-// Abstract base class for RocksDB built-in filter policies.
+// Base class for RocksDB built-in filter policies. This can read all
+// kinds of built-in filters (for backward compatibility with old
+// OPTIONS files) but does not write filters.
```

Comment:
Is "write filter" the same as "build filter"? I am more used to using the word "build" led by "FilterBitsBuilder". Is there a technical reason to use "write" instead of "build"?

Reply:
Good point. I'll use consistent wording.